sdk/keyvault/Azure.Security.KeyVault.Certificates/src/CertificatePolicy.cs

```diff
@@ -122,8 +122,9 @@ internal CertificatePolicy()
         }
 
         /// <summary>
-        /// Gets a new <see cref="CertificatePolicy"/> suitable for self-signed certificate requests.
-        /// You should change the <see cref="Subject"/> before passing this policy to create a certificate.
+        /// Gets a new <see cref="CertificatePolicy"/> suitable for self-signed certificate requests
+        /// with the <see cref="Subject"/> "CN=DefaultPolicy". To change the Subject, create a new instance
+        /// using one of the constructors.
```

Comment:
Would it make sense to reference the relevant ctors with `<see cref`s?

Reply:
All of them are relevant, so I don't think it really adds benefit (there's also no other way to create a new instance in .NET).